src/huntsman/pocs/states/huntsman/observing.py

```diff
@@ -16,19 +16,15 @@ def on_enter(event_data):
 
         wait_time = 0.
         while not all([event.is_set() for event in camera_events.values()]):
```

Comment:
Isn't it better using `panoptes.pocs.time.wait_for_events` here?

Reply:
@JAAlvarado-Montes Yes!

Reply:
Although I think we can leave this to a more generic cleanup of the `observe` function, which IMHO should/could be blocking.

Reply:
OK, sounds good!